docs/charts/radar.md

```diff
@@ -165,8 +165,7 @@ The interaction with each point can be controlled with the following properties:
 
 ## Scale Options
 
-The radar chart supports only a single scale. The options for this scale are defined in the `scale` property.
-The options for this scale are defined in the `scale` property, which can be referenced from the [Linear Radial Axis page](../axes/radial/linear).
+The radar chart supports only a single scale. The options for this scale are defined in the `scale` property, which can be referenced from the [Linear Radial Axis page](../axes/radial/linear).
```

Comment:
This might be mostly not accurate. Its not configured in the `scale` property anymore, but in `scales` object (as in the example below)